packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js

```diff
@@ -17,6 +17,32 @@ export const INSTANCE_READ_MORE_GUIDE = {
       'https://docs.ovh.com/gb/en/public-cloud/attaching-pci-floating-ip-to-instance/',
   },
 };
+export const WINDOWS_PRIVATE_MODE_LICENSE_GUIDE = {
+  en_GB:
+    'https://docs.ovh.com/gb/en/public-cloud/activate-windows-licence-private-mode-instance/',
+  en_CA:
+    'https://docs.ovh.com/ca/en/public-cloud/activate-windows-licence-private-mode-instance/',
+  en_US:
+    'https://docs.ovh.com/us/en/public-cloud/activate-windows-licence-private-mode-instance/',
+  en_AU:
+    'https://docs.ovh.com/au/en/public-cloud/activate-windows-licence-private-mode-instance/',
+  fr_FR:
+    'https://docs.ovh.com/fr/public-cloud/activate-windows-licence-private-mode-instance/',
+  fr_CA:
+    'https://docs.ovh.com/ca/fr/public-cloud/activate-windows-licence-private-mode-instance/',
+  de_DE:
+    'https://docs.ovh.com/de/public-cloud/activate-windows-licence-private-mode-instance/',
+  es_ES:
+    'https://docs.ovh.com/es/public-cloud/activate-windows-licence-private-mode-instance/',
+  it_IT:
+    'https://docs.ovh.com/it/public-cloud/activate-windows-licence-private-mode-instance/',
+  pt_PT:
+    'https://docs.ovh.com/pt/public-cloud/activate-windows-licence-private-mode-instance/',
+  ASIA:
+    'https://docs.ovh.com/asia/en/public-cloud/activate-windows-licence-private-mode-instance/',
+  DEFAULT:
+    'https://docs.ovh.com/gb/en/public-cloud/activate-windows-licence-private-mode-instance/',
+};
```

Comment:
**nitpick:** Maintain the banner based on the User Sub and not the user selected language.

Reply:
How do you manage url like fr_CA, en_CA, en_GB, en_US, en_AU if you use sub ?

Reply:
QC:
CA:
GB:
US:
AU:

Reply:
Done